src/plugins/intel_cpu/src/compiled_model.h

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
nit: maybe we can use `.hpp` extension for headers instead of `.h`

Reply:
Most of header files in cpu plugins adopt ".h" extension, maybe we need another PR to change them?